orbit/pkg/packaging/wix/wix.go

```diff
@@ -71,6 +78,25 @@ func Heat(path string, native bool, localWixDir string) error {
 	return nil
 }
 
+func darwinWineExecutable() (string, error) {
+	cmdOut, err := exec.Command("wine", "--version").Output()
```

Comment:
Does wine v9 contain the two binaries? I didn't know that, sorry.
Are they the same binary?

Reply:
Wine 9.0 ships with `wine` and `wine64` on ARM64. They are different binaries but both have the same version number, and since `wine` is available in both 9.0 and 10.0 it's safe to check.

Reply:
Gotcha. So, do we need the change I requested? Maybe your original PR was enough?

Reply:
No I think it's still a good idea, I believe `wine` in version 9.0 runs in 32-bit mode, where `wine` in 10.0 runs in 64-bit mode

Reply:
Which is probably why they removed `wine64`

Reply:
But to be honest I never actually checked that, I just assumed because we use `wine64`, it's possible there's no difference